### ISSUE TITLE
Update GRE figure spacing

### ISSUE THREAD

**Issue by dpapp86** [shimming-toolbox/rf-shimming-7t#102]
@agahkarakuzu 
@mathieuboudreau 

Sorry, guys. Some last-minute changes for spacing.

In the GRE figure field (the one created here #100 ), change the line:
```
#Create a figure with multiple subplots
fig, axes = plt.subplots(2, 4, figsize=(8, 5))
```

TO:

```
#Create a figure with multiple subplots
fig, axes = plt.subplots(2, 4, figsize=(8, 5.3))
```

And change

```
# Delete unneeded first subplot
axes[0].set_axis_off()
plt.tight_layout()
plt.subplots_adjust(wspace=0.1, hspace=0.05, right=0.9) 
plt.show()
```

TO

```
# Delete unneeded first subplot
axes[0].set_axis_off()
plt.tight_layout()
plt.subplots_adjust(wspace=0.1, hspace=0, right=0.9) 
plt.show()
```

Expected output (OBS! the vertical spacing between the rows is bigger than before)

![image](https://github.com/shimming-toolbox/rf-shimming-7t/assets/80138067/37234b0b-7bfa-4b3c-b521-9cf6c451955d)

**Comment by mathieuboudreau** [comment]
@dpapp86 this is the output - looks ok to you?

![figure_3-3](https://github.com/shimming-toolbox/rf-shimming-7t/assets/1421029/60780823-ccc0-45da-b10d-9e36414b8ddf)

**Comment by dpapp86** [comment]
@mathieuboudreau Yes

Thank you!

**Comment by mathieuboudreau** [comment]
Resolved in b234c5b
